pdr_backend/cli/cli_module.py

```diff
@@ -301,3 +302,14 @@ def do_arima_plots(args, nested_args=None):
         nested_override_args=nested_args,
     )
     arima_dash(ppss, args.debug_mode)
+
+
+@enforce_types
+# pylint: disable=unused-argument
+def do_predictoor_dashboard(args, nested_args=None):
+    ppss = PPSS(
```

Comment:
it looks like you're using the implicit PPSS. Can you use it directly from args.PPSS?